src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java

```diff
@@ -30,6 +30,10 @@ public static void main(String[] args) {
     static String getPlayingTime(int playingTimeInSeconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int day = playingTimeInSeconds / 86400;
+        int hour = playingTimeInSeconds % 86400 / 3600;
+        int minutes = playingTimeInSeconds % 86400 % 3600 / 60;
+        int seconds = playingTimeInSeconds % 86400 % 3600 % 60;
+        return day+" "+hour+":"+minutes+":"+seconds;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"